src/js/containers/covid19/Covid19Container.jsx

```diff
@@ -175,7 +175,7 @@ const Covid19Container = () => {
             </StickyHeader>
             <LoadingWrapper isLoading={isLoading}>
                 <main id="main-content" className="main-content usda__flex-row">
-                    <div className="sidebar__wrapper">
+                    <div className="sidebar__content">
```

Comment:
```suggestion
                    <div className="sidebar">
```

Reply:
Sorry my previous comment wasn't super clear - the block (`sidebar`) should wrap the element (`sidebar__content`) ([BEM docs here](http://getbem.com/introduction/))